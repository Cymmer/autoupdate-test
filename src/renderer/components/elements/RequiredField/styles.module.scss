@import '../../../scss/variables';

.RequiredField {
  color: $red-400;
  font-weight: bold;
}
