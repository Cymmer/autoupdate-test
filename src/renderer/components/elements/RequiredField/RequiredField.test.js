import React from 'react';
import { shallow } from 'enzyme';
import RequiredField from './index';

describe('RENDERING - RequiredField', () => {
  it('should render with default props', () => {
    const component = shallow(<RequiredField />);
    expect(component).toMatchSnapshot();
  });

  it('should render the placeholder', () => {
    const component = shallow(<RequiredField placeholder="Test" />);
    expect(component).toMatchSnapshot();
  });
});
