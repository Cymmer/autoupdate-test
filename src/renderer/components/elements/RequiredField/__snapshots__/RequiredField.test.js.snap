// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RENDERING - RequiredField should render with default props 1`] = `
<Fragment>
  <span
    className="RequiredField"
  >
    *
  </span>
</Fragment>
`;

exports[`RENDERING - RequiredField should render the placeholder 1`] = `
<Fragment>
  Test
  <span
    className="RequiredField"
  >
    *
  </span>
</Fragment>
`;
